shared/src/packages/shifts/libs/enums/shift-api-path.enum.ts

```diff
@@ -0,0 +1,7 @@
+const ShiftsApiPath = {
+  ROOT: '/',
+  $ID: '/:id',
+  DRIVER_$ID: '/driver',
```

Comment:
```suggestion
  DRIVER_$ID: 'driver/:driverId',
```
I don't like concatenation of strings for  building a path how it's now, replace it